### PR TITLE
Add general polish

### LINKED ISSUE #725: General polish items
**Button block**

- [x] Needs to be responsive with the link on the side

**Code**

- [x] Maybe make the inline code block a bit taller

**Lists**

- [x] Paddings and margins need love

**Switcher**

- [x] Items inside the switcher should be full width, not inline-block

**Blocks**

- <del>We should satisfyingly animate when a block is split out, or created</del>

### PR DESCRIPTION
**Pull request by jasmussen**
This closes #725, because I just split out the "block animation" bit in a separate ticket. 

Lots of little polish in this PR:

- Placeholder color in title fix
- Text trimming in switcher and elsewhere
- List style polish
- Code boundary polish
- Button block polish

### CODE REVIEW

File: blocks/library/button/index.js [hunk 1]
```diff
@@ -74,7 +74,7 @@ registerBlock( 'core/button', {
 			<span className="blocks-button" title={ title }>
 				<Editable
 					tagName="span"
-					placeholder={ wp.i18n.__( 'Write some text…' ) }
+					placeholder={ wp.i18n.__( 'Write label…' ) }
```
Comment (iseulde) [inline blocks/library/button/index.js]: Do you want to add the min-width here? :)

File: blocks/editable/style.scss [hunk 2]
```diff
@@ -28,7 +28,7 @@
 	}
 
 	code {
-		padding: 0 2px;
+		padding: 2px;
 		margin: 0 -2px;
```
Comment (iseulde) [inline blocks/editable/style.scss]: Just a suggestion... Could we try to remove this negative magic to normalise spacing? I know links have it, but that's only on select.

File: blocks/library/button/index.js (hunk 1 above)
Comment (jasmussen) [inline blocks/library/button/index.js]: I can —  but I'm wondering —  is there a different way? It kind of feels wrong to add a min-width to the button as a whole, just so the placeholder text will be visible. 

screen shot 2017-05-11 at 12 23 58

Especially when this gets translated to German which is 33% longer, the  min width would have to be adjusted again. Is there a way the placeholder can provide the min width?

Comment (iseulde) [inline blocks/library/button/index.js]: Yeah, that's what I mean with my comment in the other PR. Will have a look, atm I see no easy fix. I tried without the position absolute, and also with :after, but that doesn't really work well. This is a sting issue really. :)

File: blocks/editable/style.scss (hunk 2 above)
Comment (jasmussen) [inline blocks/editable/style.scss]: Yes, that feels better!

File: blocks/library/button/index.js (hunk 1 above)
Comment (jasmussen) [inline blocks/library/button/index.js]: > This is a sting issue really. :)

"Sting issue"?

I'm sure we'll find a solution, even if we have to do it through clever labels and liberal min-widths.

Comment (iseulde) [inline blocks/library/button/index.js]: Sorry, styling.

**Review by aduth — APPROVED** [review]

**Pull request merged**